### ISSUE TITLE
Keyboard/gamepad navigation

### ISSUE THREAD

**Issue by PiKeyAr** [X-Hax/SA-Mod-Manager#249]
At the moment the Mod Manager cannot be navigated with TAB+arrow keys. I suggest implementing keyboard navigation for the following reasons:
1. Almost all Windows programs can be navigated with a keyboard, and not being able to do it in the Manager feels counter-intuitive.
2. Some areas in the Game Config tab (resolution boxes in Graphics settings) do respond to TAB but most other UI elements don't. The The Mods ListView-like control can be navigated with letter keys, but not the Codes one. Inconsistent behavior makes it seem incomplete or buggy.
3. Feature parity with legacy Mod Managers.
4. This would provide the necessary groundwork for implementing gamepad navigation in the future.
